packages/wdio-appium-service/src/@types/appium-service.d.ts

```diff
@@ -0,0 +1,21 @@
+
+type ArgValue = string | number | boolean | null
```

Comment:
from Appium documentation http://appium.io/docs/en/writing-running-appium/server-args/index.html I can see that there is only one argument could be an object `--default-capabilities`. 
Previous implementation were coercing it to `[object Object]`.  Handling of an object could be added as scope of separate PR if needed.

Reply:
`default-capabilities` should be from type string and passed along as such to Appium

Reply:
So, does it mean that we can keep 'object' type forbidden for `default-capabilities` and leave string transformation to user?

Reply:
ah I see now, we can keep as is, it is not changing any behavior 👍